public/models/interfaces.ts

```diff
@@ -234,14 +234,23 @@ export type AnomalyData = {
   plotTime?: number;
   entity?: EntityData[];
   features?: { [key: string]: FeatureAggregationData };
-  aggInterval?: string;
+  contributions?: { [key: string]: FeatureContributionData };
+  aggInterval?: string; 
 };
 
 export type FeatureAggregationData = {
   data: number;
+  name?: string;
   endTime: number;
   startTime: number;
   plotTime?: number;
+  attribution?: number;
+  expectedValue?: number;
```

Comment:
is this also shown on the chart as part of this PR? Or just for saving the data for future UI improvements?

Reply:
this also shown on the chart. See `LineSeries` rendering in FeatureChart.